microstack/review-single-node-quickstart/single-node-quickstart.md

```diff
@@ -0,0 +1,76 @@
+This tutorial shows how to install MicroStack in the simplest way possible. It will deploy an **OpenStack 2023.1** (Antelope) cloud.
+
+The cloud will only allow access to its VMs from the local host. To enable access from any host on your network, follow the [Single-node guided](/t/35765) tutorial instead.
+
+## Requirements
+
+You will need a single machine whose requirements are:
+
+- physical or virtual machine running Ubuntu 22.04 LTS
+- a multi-core amd64 processor ideally with 4+ cores
+- a minimum of 16 GiB of free memory
+
+[note type="caution"]
+**Caution:** Any change in IP address of the local host will be detrimental to the deployment. A virtual host will generally have a more stable address.
+[/note]
+
+## Deploy the cloud
+
+### Install the openstack snap
+
+Begin by installing the openstack snap:
```

Comment:
Yeah, my years-old style guide leaves charm names unformatted and I applied that to snap names. I tried to avoid confusion when referring to application names and unit names, which I do format. But I agree that there is something unsatisfactory using these names as-is. 🤔